examples/example.py

```diff
@@ -24,14 +24,15 @@
 
 
 metadata = {
-    "version": "0.2.0",
+    "version": "0.3.0",
```

Comment:
If you're upgrading to 0.3.0 you also need to update the readme

Reply:
I also think we should update the PR title to say something like:
```
0.3.0 release: explicit optional orientation field for polygon winding
```

Reply:
Done